### DISCUSSION TITLE
Support for trle custom level.

### DISCUSSION THREAD

**Discussion by Damocle92** [LostArtefacts/TRX#2112] [Q&A] [answered]
Thank you for the awesome work.
Old version of tomb1main play nice with trle levels like Nordic-adventure, Sabatu-Tr1 ecc.
On recent version of TR1X some custom level are crashing on load, i wonder why and if is possible to fix or have some compatibility options for old custom levels.
For example i cannot start the second level of Feder-oasis, i attach the log.
[TR1X.log](https://github.com/user-attachments/files/18234982/TR1X.log)

**Comment by lahm86** [comment, marked as answer]
Thanks for reporting this @Damocle92. I've raised an issue to investigate this - #2114.

**Comment by aredfan** [comment]
Fixed by Lahm in #2120.

**Edit:** You can try out the fix in the latest [Development snapshot](https://github.com/LostArtefacts/TRX/releases/tag/latest).
